lib/Experian/IDAuth.pm

```diff
@@ -334,15 +342,10 @@ sub _get_result_proveid {
     }
 
     # check if client is age verified
-    my $kyc_dob = $kyc_summary->findvalue('DateOfBirth/Count') || 0;
     my $cr_total = $credit_reference->findvalue('TotalNumberOfVerifications')
         || 0;
     $decision->{num_verifications} = $cr_total;
 
-    if ($kyc_dob and $cr_total) {
```

Comment:
Seems like the other changes could be avoided if this line is changed to: if ($nameCount + $dobCount >= 3)